src/proguard/proguard.targets

```diff
@@ -24,6 +30,11 @@
   </Target>
   <Target Name="_CleanProGuard"
       AfterTargets="Clean">
+    <SetEnvironmentVar
```

Comment:
It *looks* like that the primary reason for `<SetEnvironmentVar/>` is for `ant`. Would it be better to have an `<Ant/>` task which calls `ant` and also calls `Environment.SetEnvironmentVariable()`?

```xml
<Ant
    Command="-f buildscripts\build.xml clean"
    ToolPath="$(AntToolPath)"
    ToolExe="$(AntToolExe)"
    JavaSdkPath="$(JavaSdkDirectory)"    
/>
```